src/Api/Droplet.php

```diff
@@ -27,14 +27,24 @@ class Droplet extends AbstractApi
      */
     public function getAll()
     {
-        $droplets = $this->adapter->get(sprintf('%s/droplets?per_page=%d', self::ENDPOINT, PHP_INT_MAX));
-        $droplets = json_decode($droplets);
-
-        $this->extractMeta($droplets);
-
-        return array_map(function ($droplet) {
+        $thereAreNewResults = true;
+        $results = [];
+        $url = sprintf('%s/droplets?per_page=%d', self::ENDPOINT, PHP_INT_MAX);
```

Comment:
I wonder if setting per_page to `PHP_INT_MAX` isn't kind of abusive of their API. Maybe something like 1000?